azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachine.java

```diff
@@ -913,6 +904,14 @@
              * @return the next stage of the definition
              */
             WithLinuxRootPasswordOrPublicKeyManagedOrUnmanaged withRootUsername(String rootUserName);
+
+            /**
+             * Specifies to remove the information of OS profile for the Linux virtual machine.
+             * It is required when creating from a specialized image.
+             *
+             * @return the next stage of the definition
+             */
+            WithLinuxCreateManagedOrUnmanaged withoutRootUsernameAndPassword();
```

Comment:
Suggestion: Maybe same method like `withNoCredential` for both windows and linux flow?

Reply:
In vm/vmss, it should be OK as we already have existing methods like:
`withoutPrimaryPublicIPAddress`: the VM should not have a public IP address.
`withoutAutoUpdate`: disables automatic updates.